### DISCUSSION TITLE
Error Messages in Slingerplayer

### DISCUSSION THREAD

**Discussion by GIBLechat** [GerryDazoo/SlingerPlayer#3] [General]
When I start Slingerplayer, I get the following:

![image](https://user-images.githubusercontent.com/124520835/221710276-34abe12d-079e-4172-b4eb-277692eb5760.png)

After clicking "OK", it plays just fine.

When I exit Slingerplayer, I get the following:

![image](https://user-images.githubusercontent.com/124520835/221710401-b3823bf5-751d-4ff2-b352-3bc635d8aea2.png)

Regards,

**Comment by martysm** [comment]
I get the duplicates error on Win10 also on startup.  I do NOT get the memory leak one on exit.

Regards all.

**Reply by GerryDazoo** [reply]
Delete all the saved URLs and let slingerplayer repopulate the list.

**Comment by GIBLechat** [comment]
Gerry,

 

Thanks again. I tried your suggestion and no difference.

 

Regards,

**Comment by martysm** [comment]
me too.  I deleted it from the sites.dat file and I got the dupes not allowed error msg and, after clearing it, the app opened with a blank URL field(http://).
I pasted it there and it was fine from that point.  THe app did write the URL back to the file.
